manager/apis/app/v1beta1/interface_types.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1beta1
+
+import (
+	"fybrik.io/fybrik/pkg/model/taxonomy"
+)
+
+// connection types and formats used in tests and mockup connectors
+const (
```

Comment:
These definitions are not used in v1alpha1 and should be removed.

Reply:
OK

Reply:
The same comment about location of the constants.